test/embag_test_python.py

```diff
@@ -97,12 +97,12 @@ def testROSMessages(self):
                     self.assertNotEqual(v, 0)
 
             if topic == '/base_pose_ground_truth':
-                self.assertEqual(msg.header.seq, base_pose_seq)
+                self.assertEqual(msg.get('header').get('seq'), base_pose_seq)
```

Comment:
Typically, when I see `get` in python, I expect it to function like "find a value in a dict, if that can't be found, default to some specified value or `None` (if default is not specified)". Does our code follow the same paradigm?

Reply:
We don't support that paradigm (and we didn't before this change either). I think I'd rather leave that up to another PR to implement though, and leave this one as just a bugfix for the existing functionality that had a regression.

Reply:
Fair enough